src/pages/About/index.tsx

```diff
@@ -149,10 +149,47 @@ const GetStarted = styled.div`
   margin-bottom: 100px;
 `
 
+const CARDS = [
+  {
+    to: '/swap',
```

Comment:
Should these be hashed routes (eg `#/swap`)?

Reply:
gonna fix all the routing in another pr